<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>244</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="year_widget">
   <property name="geometry">
    <rect>
     <x>245</x>
     <y>37</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>109</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>17</y>
     <width>651</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>145</y>
     <width>20</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="year_hun">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>년도</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>138</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="marks_1">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>120</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>간 단 요 약</string>
   </property>
  </widget>
  <widget class="QLabel" name="amount_1">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>110</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>헌금액</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="hun_date">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>36</y>
     <width>65</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>헌금일자</string>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>72</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>기타소득 명칭</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="marks_widget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>371</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="hun_name_widget">
   <property name="geometry">
    <rect>
     <x>354</x>
     <y>71</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>100</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>7</y>
     <width>56</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>173</y>
     <width>621</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="amount_widget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>148</y>
     <width>171</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="huncancel_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>190</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="hunsave_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>헌금계정 수정저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QDateEdit" name="hun_date_widget">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>35</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDate|Qt::ImhPreferNumbers</set>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::DaySection</enum>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>1</month>
     <day>1</day>
    </date>
   </property>
  </widget>
  <widget class="QLabel" name="month_hun">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>37</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>월</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="month_widget">
   <property name="geometry">
    <rect>
     <x>328</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="week_hun">
   <property name="geometry">
    <rect>
     <x>368</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>몇번째 주</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="week_widget">
   <property name="geometry">
    <rect>
     <x>448</x>
     <y>36</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="Bankincome_check">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="locale">
    <locale language="Kinyarwanda" country="Rwanda"/>
   </property>
   <property name="text">
    <string>통장예입</string>
   </property>
   <property name="autoRepeatDelay">
    <number>302</number>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>127</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>(단위:천원)</string>
   </property>
  </widget>
  <widget class="QLabel" name="hun_name_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>69</y>
     <width>65</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>회계구분</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_name_combo_widget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>year_widget</tabstop>
  <tabstop>month_widget</tabstop>
  <tabstop>week_widget</tabstop>
  <tabstop>gubun_name_combo_widget</tabstop>
  <tabstop>hun_name_widget</tabstop>
  <tabstop>amount_widget</tabstop>
  <tabstop>marks_widget</tabstop>
  <tabstop>hunsave_button</tabstop>
  <tabstop>huncancel_button</tabstop>
  <tabstop>Bankincome_check</tabstop>
  <tabstop>hun_date_widget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>huncancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>488</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hunsave_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>file_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>153</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>file_save()</slot>
  <slot>file_close()</slot>
 </slots>
</ui>
